snap/snapcraft.yaml

```diff
@@ -8,8 +8,8 @@ type: app
 base: core20
```

Comment:
I guess the base should be `core22` if this branch is for 22? and in this case, I think `run-on` needs to be changed to `build-for`

Reply:
done, fix committed